axelor-human-resource/src/main/java/com/axelor/apps/hr/service/leave/LeaveRequestMailService.java

```diff
@@ -25,9 +25,17 @@
 public interface LeaveRequestMailService {
   Message sendCancellationEmail(LeaveRequest leaveRequest) throws AxelorException;
 
+  void sendCancellationEmailAndTrace(LeaveRequest leaveRequest) throws AxelorException;
+
   Message sendConfirmationEmail(LeaveRequest leaveRequest) throws AxelorException;
 
+  void sendConfirmationEmailAndTrace(LeaveRequest leaveRequest) throws AxelorException;
+
   Message sendValidationEmail(LeaveRequest leaveRequest) throws AxelorException;
 
+  void sendValidationEmailAndTrace(LeaveRequest leaveRequest) throws AxelorException;
+
   Message sendRefusalEmail(LeaveRequest leaveRequest) throws AxelorException;
+
+  void sendRefusalEmailAndTrace(LeaveRequest leaveRequest) throws AxelorException;
```

Comment:
I don't understand why do we need to create these methods, can't we just catch the exception in LeaveRequestRestController to trace it ? Either way please refactor this, as it is duplicating code.